docs/loadflow/loadflow.md

```diff
@@ -17,6 +17,9 @@ equal to zero and $1$. In case of a branch with voltage or phase control, the $\
 ![Pi model](pi-model.svg){class="only-light"}
 ![Pi model](pi-model-dark-mode.svg){class="only-dark"}
 
+Note that transformers are modeled with a single admittance on one side (conductance $G$ and susceptance $B$). This admittance can be splitted into two parts (on both sides) with the [PowSyBl LoadFlow `twtSplitShuntAdmittance` Parameter](inv:powsyblcore:*:*#simulation/loadflow/configuration).
```

Comment:
grammar: `can be splitted` -> `can be split`

Reply:
Thanks, I forgot my irregular verbs :eyes: